cmd/clusterctl/pkg/client/delete.go

```diff
@@ -70,12 +70,17 @@ func (c *clusterctlClient) Delete(options DeleteOptions) error {
 			}
 
 			// Check the provider/namespace tuple actually matches one of the installed providers.
+			found := false
```

Comment:
The changes in this file seem like it should be in a separate PR.

Reply:
Yep, testing logs I found some small bugs and I fixed them.